Formula/minio.rb

```diff
@@ -2,9 +2,9 @@ class Minio < Formula
   desc "Amazon S3 compatible object storage server"
   homepage "https://github.com/minio/minio"
```

Comment:
Should this be updated to https://min.io?

Reply:
Could call, did not know that before. :)